### PR TITLE
monitoring reverse proxies

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adding content for reverse proxies in the monitoring section.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/11077

### CODE REVIEW

File: modules/administration/pages/monitoring.adoc [hunk 1]
```diff
@@ -313,3 +319,26 @@ You can also proxy requests through the network boundary.
 Tools like PushProx deploy a proxy and a client on both sides of the network barrier and allow Prometheus to work across network topologies such as NAT.
 
 For more information on PushProx, see https://github.com/RobustPerception/PushProx.
+
+
+
+=== Reverse Proxy Set Up
```
Comment (keichwa) [inline modules/administration/pages/monitoring.adoc:325]:
```suggestion
=== Reverse Proxy Setup
```

Comment (Loquacity) [reply modules/administration/pages/monitoring.adoc:325]: Heh, good spot. In my defence, I originally had "Set Up Reverse Proxy" :wink:

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Heh, good spot. In my defence, I originally had "Set Up Reverse Proxy" 😉

Ok, then it is just a copypasta issue ;)

-- 
Karl Eichwalder

File: modules/administration/pages/monitoring.adoc [hunk 2]
```diff
@@ -313,3 +319,26 @@ You can also proxy requests through the network boundary.
...
+
+Prometheus fetches metrics using a pull mechanism, so the server must be able to establish TCP connections to monitored clients.
+If you cannot open these ports directly in your firewall, you can use reverse proxies to establish a connection.
+
+// Probably a diagram here. --LKB 2020-08-11
+
+
+
+.Procedure: Installing Prometheus with Reverse Proxies
+. In the {productname} {webui}, open the details page of the system where Prometheus is to be installed, and navigate to the [guimenu]``Formulas`` tab.
```
Comment (witekest) [inline modules/administration/pages/monitoring.adoc:335]:
```suggestion
. In the {productname} {webui}, open the details page of the system to be monitored, and navigate to the [guimenu]``Formulas`` tab.
```

File: modules/administration/pages/monitoring.adoc [hunk 3]
```diff
@@ -288,9 +290,13 @@ image::monitoring_configure_formula.png[scaledwidth=80%]
 Monitoring formulas can also be configured for System Groups, by applying the same configuration used for individual systems inside the corresponding group.
 ====
 
+// The WebUI says that we describe the available arguments in the exporters formula, but I don't see it. Might be best to put it in the Salt Guide, then xref from here. --LKB 2020-08-11
```
Comment (witekest) [inline modules/administration/pages/monitoring.adoc:293]: The help string says: _Please refer to the documentation for available options._. These are different for every exporter. I'm not sure if that's in scope of product documentation. @renner , could you comment on this please?

Comment (Loquacity) [reply modules/administration/pages/monitoring.adoc:293]: Let's address this in https://github.com/SUSE/spacewalk/issues/12001 so we can merge this one now.

**Comment by Loquacity** [comment]
:heavy_check_mark: backported to `manager-4.1` cce56e7eff7021041873ee63a08959b919e636e1

**Pull request merged**